mypy/types.py

```diff
@@ -818,6 +843,30 @@ def max_possible_positional_args(self) -> int:
         blacklist = (ARG_NAMED, ARG_NAMED_OPT)
         return len([kind not in blacklist for kind in self.arg_kinds])
 
+    def formal_arguments(self, include_star_args: bool = False) -> Iterator[FormalArgument]:
+        """Yields the formal arguments corresponding to this callable, ignoring *arg and **kwargs.
+
+        To handle *args and **kwargs, use the 'callable.var_args' and 'callable.kw_args' fields,
+        if they are not None.
+
+        If you really want to include star args in the yielded output, set the
+        'include_star_args' field to 'True'."""
```

Comment:
> set field -> pass argument

It seems you missed this comment. Or you think "set field" is an OK formulation for functions?